packages/embeds/embed-core/src/embed.ts

```diff
@@ -466,6 +468,10 @@ class CalApi {
       el.dataset.calLink = calLink;
       el.dataset.calNamespace = this.cal.namespace;
       el.dataset.calOrigin = calOrigin ?? "";
+      if (config) {
+        el.dataset.calConfig = JSON.stringify(config);
```

Comment:
Because floating button popup uses Element Click popup logic only, adding this data attribute is enough.